lib/Redmine/Api/Search.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Redmine\Api;
+
+/**
+ * Searching
+ *
+ * @see   http://www.redmine.org/projects/redmine/wiki/Rest_Search
+ *
+ * @author Javier Sanahuja

```

Comment:
please don't add `@author` tags! :) your name will show up in the contributors' section ;)